doc/integrator/backend_qgis.rst

```diff
@@ -53,6 +53,7 @@ In the ``vars_<project>.yaml`` file, define:
         ...
         mapserverproxy:
             default_ogc_server: <ogc server name>
+            mapserv_url: http://your.qgis.server/ows
```

Comment:
This parameter does no longer exists..., he is replaced by `default_ogc_server`...